monitoring/config/alert.rules.tmpl

```diff
@@ -0,0 +1,80 @@
+groups:
+- name: High CPU
+  rules:
+%{ for app in apps ~}
+  - alert: High-CPU-${app}
+    expr: avg by ( app ) (cpu{app=~"${app}"}) > 50
+    for: 5m
+    annotations:
+      summary:     ${app} High CPU Alert
+      dashboard:   ${grafana_dashboard_url}&var-Applications=${app}
+      description: "CPU usage of at least 50% for more than 5 minutes (current value: {{ $value }})"
```

Comment:
Maybe remove the hard coded 50% here so if we change later we only change in one place?
for example: "High CPU usage for more than 5 minutes (current value: {{ $value }})"